core/procedures/observable_procedure_model.ts

```diff
@@ -56,8 +58,13 @@ export class ObservableProcedureModel implements IProcedureModel {
    * Pass null to represent a procedure that does not return.
```

Comment:
Can you add the information about how this isn't supported to the TsDoc for this function?  From the lens of someone viewing the reference doc, they should be able to tell that they shouldn't call this method (on this class) and need to implement their own.